.scripts/test_all.sh

```diff
@@ -26,6 +26,30 @@ else
   go get -u github.com/hashicorp/consul/...
 fi
 
+install_etcd_pkgs() {
+  go get -u github.com/coreos/etcd
```

Comment:
I believe the repository has moved to https://github.com/etcd-io/etcd, hasn't it?

Reply:
Correct, but I believe the `release` branches at least when I was working on it - were still referencing the old `coreos/etcd` and therefore nothing would work if I changed it to `etcd-io/etcd`. Perhaps that is changed on the latest `master` branch, but I haven't checked back.

Reply:
Ok, makes sense that it just changed recently. Would you mind taking a look?

Reply:
I believe the current `v3.3.9` release branch still references `coreos/etcd`

examples: 
https://github.com/etcd-io/etcd/blob/fca8add78a9d926166eb739b8e4a124434025ba3/clientv3/cluster.go
https://github.com/etcd-io/etcd/blob/fca8add78a9d926166eb739b8e4a124434025ba3/clientv3/client.go

Reply:
Ok, then let's keep `coreos/etcd` and try it out with gopkg.in